### PR TITLE
git-stash: add French translation

### PR DESCRIPTION
**Pull request by Hyftar**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

This PR adds a French translation of the `git stash` command.

### CODE REVIEW

File: pages.fr/common/git-stash.md
Comment (schneiderl) [inline pages.fr/common/git-stash.md:8]: Is this example parameter already translated? I've quickly thrown it into a translator and it seems to be already translated but also it looks exactly like English so just making sure.

Comment (Hyftar) [reply pages.fr/common/git-stash.md:8]: Oops, my bad I completely missed the parameter. It is indeed still English.

**Review by navarroaxel — APPROVED** [review]
LGTM @Hyftar, please don't `push --force` to the PR's branch, it's harder to us to review your changes and track the suggestions. We squash before merge. :grin:

File: pages.fr/common/git-stash.md
Comment (sbrl) [inline pages.fr/common/git-stash.md:6]: Any particular reason `*stash*` is in italics here? Just curious, since this isn't the case in the original page.

I'm unsure as to whether this will be rendered correctly - I can't remember whether we've done this elsewhere or not (I don't remember any specific examples off-hand).

Comment (Hyftar) [reply pages.fr/common/git-stash.md:6]: Because there isn't a universal French equivalent for stash so I used the English word but since it's still in English I have to put it in italics.

Comment (sbrl) [reply pages.fr/common/git-stash.md:6]: Ah, I see. This isn't technically against the client spec, so this should be ok. I'd just like another opinion  before we go ahead though, if you don't mind waiting on this PR for a little while.

**Review by schneiderl — APPROVED** [review]

File: pages.fr/common/git-stash.md
Comment (schneiderl) [reply pages.fr/common/git-stash.md:6]: @sbrl If I'm not completely mistaking, we already do this in some pt_BR pages so I believe this is fine.

Comment (sbrl) [reply pages.fr/common/git-stash.md:6]: Oh, sounds good! Just wanted to check. In that case, it's fine as-is.

**Review by sbrl — APPROVED** [review]

**Pull request merged**
